src/interface/widgets/SettingsWidget.cpp

```diff
@@ -27,7 +27,6 @@ SettingsWidget::SettingsWidget(QWidget *parent) {
     grsimPort->setValue(SETTINGS.getRobothubSendPort());
     grsimSettingsWidgetLayout->addWidget(grsimPort);
     grsimSettingsGroup->setLayout(grsimSettingsWidgetLayout);
-    QObject::connect(grsimPort, SIGNAL(textChanged(QString)), this, SLOT(changeGrSimPort(int)));
```

Comment:
Does this mean we cannot change the grSim port anymore?

Reply:
That feature never worked anyway -> QT gave an error that the Signal did not exist